src/components/Badge/styled.tsx

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+import { em, transparentize } from 'polished';
+
+export const variants = ['success', 'warning', 'danger', 'primary'] as const;
+export type Variant = typeof variants[number];
+
+export const transparents = ['success', 'warning', 'danger', 'primary', 'buy', 'sell'] as const;
+export type Transparent = typeof transparents[number];
+
+const success = css`
+  background: ${({ theme }) => transparentize(0.9, theme.honeycomb.color.success.active)};
```

Comment:
Just to double check, are you sure `theme.honeycomb.color.success.active` is the right colour and not `theme.honeycomb.color.success.normal`?